cli/src/main/java/com/liferay/blade/cli/command/UpdateCommand.java

```diff
@@ -254,10 +254,9 @@ public void execute() {
 				bladeCLI.out("Updating from: " + url);
 
 				if (BladeUtil.isWindows()) {
-					bladeCLI.out(
-						"blade update cannot execute successfully because of Windows file locking.  Please use the " +
-							"following command:");
-					bladeCLI.out("\tjpm install -f " + url);
+					Runtime runtime = Runtime.getRuntime();
+
+					runtime.exec("cmd /c start \"\" jpm install -f " + url + " && exit");
```

Comment:
I know why this works... download time.  It takes jpm some time to download the new binary of blade jar, and in the meantime, the java process has time to exit because of the way you are invoking the shell "running in background and then exiting the shell".